### PR TITLE
Aidantran/no render app fix

### PR DESCRIPTION
**Pull request by AidanTran**
Trying to remove the necessity to mock requestAnimationFrame globally when running NoRenderApp in tests outside of certa. 
NOT CURRENTLY TESTED
Having trouble testing this as current RPC tests are run through certa.

### CODE REVIEW

**Comment by AidanTran** [comment]
@wgoehrig Is something like this what you had in mind?

File: core/frontend/src/IModelApp.ts [hunk 1]
```diff
@@ -123,6 +123,8 @@ export interface IModelAppOptions {
   renderSys?: RenderSystem | RenderSystem.Options;
   /** If present, supplies the [[UiAdmin]] for this session. */
   uiAdmin?: UiAdmin;
+  /** If present, determines whether iModelApp is a NoRenderApp */
+  noRender?: boolean;
```
Comment (pmconne) [inline core/frontend/src/IModelApp.ts:127]: Mark `@internal` please.

File: core/frontend/src/IModelApp.ts [hunk 2]
```diff
@@ -269,6 +271,10 @@ export class IModelApp {
   public static get hasRenderSystem() { return this._renderSystem !== undefined && this._renderSystem.isValid; }
   /** The [[UiAdmin]] for this session. */
   public static get uiAdmin() { return this._uiAdmin; }
+  /** If the app is a NoRenderApp
+  * @internal
+  */
+    public static get noRender() { return this._noRender; }
```
Comment (pmconne) [inline core/frontend/src/IModelApp.ts:277]: What is the motivation for this accessor? Nobody currently calls it and ideally no one will need to.

**Review by pmconne — CHANGES_REQUESTED** [review]
It looks like you may not have done a `rush build` before `rush clean`. You definitely should have no API changes except in core-frontend.
Consider reverting your extract-api commit and doing:
```
rush clean
rush build --to core-frontend
cd core/frontend
npm run extract-api
```

**Review by pmconne — COMMENTED** [review]
Did you `rush install`? We're using a new version of api-extractor -- looks like you might be using the old version.

**Review by pmconne — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**